### ISSUE TITLE
Assumptions

### ISSUE THREAD

**Issue by christianvoigt** [christianvoigt/argdown#42]
Support for assumptions in argument reconstruction is missing.
Possible syntax:

(1) ? [Flat Eartherism]: The earth is flat.

Advantages of syntax:

- only one character (no syntax-clutter)
- intuitive meaning
- no double colons
- no english words (asm, assume, assumption)
- no brackets
- no special characters that already have a different meaning in Argdown

**Comment by tobias-martin** [comment]
Dear @christianvoigt,

This would be a very neat (but not absolutely necessary) addition to argdown. I would love to be able to reconstruct argumentations which make use of assumptions for the sake of the argument (conditional proof, reductio and so on). In a far future, even an implementation for fitch-style-proofs might be possible?

Yet, I suspect, some reworking or reinterpretation of the syntax would be needed to fit in assumptions more smoothly. Since assumptions and theses just differ in illocutionary force, the bracket-notation "[...]" could be reinterpreted not to define theses but propositions. The same goes for polar questions. In this way, there is some relevance to https://github.com/christianvoigt/argdown/issues/218. Given propositions, different illocutionary acts can be introduced via illocutionary operators on propositions, e.g.: ". [p]" for "the thesis that p", ": [p]" for "the assumption that p" and "? [p]" for "the question whether p". (A star-notation for assumptions "* [p]" would be optimal since it was already established by Quine in his  _Methods of Logic_, §27. But this would collide with the overall usage of "*", am I right?)

I wonder: Do we really need an operator for assumptions in this case where we have a tool for referring to propositions (e.g. via reinterpreted "[p]"). Wouldn't this suffice? The crucial point of adding assumptions is to allow for the reference to propositions without _asserting_ them. I don't want to _assert_ the antecedent or the consequent when I argue for a conditional. Even more clearly in the case of a reductio: I would never want to _assert_ the reductio hypothesis or the absurdum!

In conclusion, what do you think of **(1)** reinterpreting the bracket notation "[...]" for the definition and reference to propositions, **(2)** introducing an illocutionary operator on propositions for the introduction of illocutionary acts (e.g. ". [p]" for theses), but **(3)** dropping such an operator in the case of assumptions, since the reference to propositions via "[p]" does already suffice?

All the best,
Tobias
